<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandDisplayChoice</class>
 <widget class="QDialog" name="DialogCommandDisplayChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display a choice...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <item>
        <widget class="QGroupBox" name="groupBoxChoices">
         <property name="title">
          <string>Choices</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="WidgetSuperTree" name="treeViewChoices">
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxOptions">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
            <item>
             <widget class="QLabel" name="labelCancelAutoIndex">
              <property name="text">
               <string>Cancel auto index:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PanelPrimitiveValue" name="panelPrimitiveCancelAutoIndex" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>238</width>
                <height>178</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSuperTree</class>
   <extends>QTreeView</extends>
   <header>widgetsupertree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandDisplayChoice</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandDisplayChoice</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
